ImageAnalysis/image_analysis/analyzers/UC_GenericLightSpecCam.py

```diff
@@ -30,7 +30,9 @@ def __init__(self,
                  calibration_0th_order_pixel: float = 1186.24,
                  minimum_wavelength_analysis: float = 150.0,
                  optimization_central_wavelength: float = 420.0,
-                 optimization_bandwidth_wavelength: float = 10.0
+                 optimization_bandwidth_wavelength: float = 10.0,
+                 spec_correction_bool: bool = True,
+                 spec_correction_files: List[str] = []
```

Comment:
Consider writing out the variable names: spectral_correction_files, corrected_image, efficiency_lower_bound, etc.  Descriptive variable names effectively self-document code at negligible cost.